src/music.ts

```diff
@@ -331,11 +329,11 @@ export class Music {
 
     let relativeNoteName = parts.root;
     if (interval > 0) {
-      for (let i = 1; i <= interval; ++i) {
+      for (let i = 1; i <= interval; i += 1) {
```

Comment:
why changing this?

Reply:
aribnb rule no use of ++ or --

Reply:
Ok, they have their conventions. But why we should follow exactly this one?

Reply:
see answer below, the reasons are documented